salt/modules/kubernetes.py

```diff
@@ -78,6 +83,21 @@ def __virtual__():
     return False, 'python kubernetes library not found'
 
 
+if salt.utils.is_windows():
```

Comment:
Oh, wait a second. This is wrong! This should be the other way around. Windows has *no* singnals.